package.json

```diff
@@ -20,8 +20,9 @@
     "exec"
   ],
   "dependencies": {
-    "gulp-util": "~2.2.14",
-    "plexer": "0.0.1"
+    "plexer": "0.0.1",
+    "plugin-error": "0.1.2",
+    "vinyl": "2.1.0"
```

Comment:
I may be missing something but I think `vinyl` should in the `devDependencies` because it's only used in the tests.

Reply:
I will update dependencies shortly. What node version should it be for the travis.yml?